runtime/runtime/src/config.rs

```diff
@@ -172,7 +172,7 @@ pub fn tx_cost(
     let maximum_depth =
         if minimum_new_receipt_gas > 0 { prepaid_gas / minimum_new_receipt_gas } else { 0 };
     let inflation_exponent =
-        u8::try_from(initial_receipt_hop + maximum_depth).map_err(|_| IntegerOverflowError {})?;
+        u8::try_from(initial_receipt_hop + maximum_depth).unwrap_or_else(|_| u8::max_value());
```

Comment:
Isn't 255 too low for the exponent cap? In the future our inflation might be <<1% per block.

Reply:
I think the idea is to limit maximum depth to some fixed amount. Right now this is done through price in gas in practice, but for parameter estimator we have u64::MAX as the limit for a number of things and so it breaks here.

Reply:
Why we can't use `gas_price` equal to 0 instead, it should be good for estimation, no?

Reply:
I am still not sure why this should be capped. If someone sends a tx that has so much gas that it overflows here then it is an invalid tx.

Reply:
> If someone sends a tx that has so much gas that it overflows here then it is an invalid tx.

The issue is that in runtime estimator we use `u64::MAX` for prepaid gas and that will cause overflow here.

Reply:
> Why we can't use gas_price equal to 0 instead, it should be good for estimation, no?

@evgenykuzyakov that doesn't fix the overflow